yafti/app.py

```diff
@@ -27,7 +27,7 @@
 
 class Yafti(Adw.Application):
     def __init__(self, cfg: Config = None, loop=None):
-        super().__init__(application_id="it.ublue.Yafti")
+        super().__init__(application_id="it.ublue.Yafti" title="Setup")
```

Comment:
I'd prefer this be set to `config.title` so that distro builders can call this whatever they'd like

Reply:
Also need a , between the two arguments

Reply:
> I'd prefer this be set to config.title so that distro builders can call this whatever they'd like

Like so? 

```python
(application_id="it.ublue.Yafti", title=config.title)
```

Reply:
Sorry, I typed that on my phone, it's `cfg` and not `config`. Otherwise, yes!

```python
(application_id="it.ublue.Yafti", title=cfg.title)
```